edit/edit.js

```diff
@@ -295,13 +295,13 @@ function acmeEventListener(event) {
     return;
   }
   let value = el.type == 'checkbox' ? el.checked : el.value;
-  let url;
-  const themeLink = document.getElementById('cm-theme');
   switch (option) {
     case 'tabSize':
       CodeMirror.setOption('indentUnit', Number(value));
       break;
+    /* eslint-disable no-case-declarations */
```

Comment:
Hmm, I've never tried wrapping a case in curly brackets before... if it works, then cool!